a3p-integration/proposals/b:wf2/README.md

```diff
@@ -0,0 +1,19 @@
+# Proposal to upgrade Smart Wallet to incarnation 2
+
+This is based on https://github.com/Agoric/agoric-3-proposals/pull/34 which tested https://github.com/Agoric/agoric-sdk/pull/8488
+
+How this differs is it's an experiment to test the proposal in a3p from within agoric-sdk
+
+It should automate these steps:
+
+```
+# whatever your checkout
+A3P=/opt/agoric/agoric-3-proposals
+cd packages/builders
+# build the proposal
+agoric run scripts/vats/upgrade-wallet-factory.js | tee run-report.txt
+# copy the proposal
+cp upgrade-wallet-factory* $A3P/proposals/a:wallet-factory/submission
+# copy the bundles built for the proposal
+cat run-report.txt | grep install-bundle | sed "s/agd tx swingset install-bundle @//" |xargs -I _ cp _ $A3P/proposals/a:wallet-factory1/submission
```

Comment:
Can we either delete run-report.txt or .gitignore it?

Reply:
The build step does delete it: https://github.com/Agoric/agoric-sdk/blob/8dbf55c4e5667e1afb3f8938b16b67db9e2b5024/packages/synthetic-chain/src/cli/build.ts#L25

I prefer this to `.gitignore` so it doesn't sit in the filesystem.